langchain4j-qdrant/src/test/java/dev/langchain4j/store/embedding/qdrant/QdrantEmbeddingStoreIT.java

```diff
@@ -29,22 +29,22 @@ class QdrantEmbeddingStoreIT extends EmbeddingStoreIT {
   private static QdrantEmbeddingStore embeddingStore;
 
   @Container
-  private static final QdrantContainer qdrant = new QdrantContainer("qdrant/qdrant:latest");
+  private static final QdrantContainer qdrant = new QdrantContainer("qdrant/qdrant:v1.11.1");
```

Comment:
Ideally this test should make sure LC4j implementation works well with Qdrant, including the latest version. So I would revert this change. It should fail as soon as possible so that we can start fixing it asap...

Reply:
Done.